pkg/service/egress.go

```diff
@@ -18,48 +21,59 @@ import (
 )
 
 type EgressService struct {
-	rpcClient   egress.RPCClient
-	store       ServiceStore
-	es          EgressStore
-	roomService livekit.RoomService
-	telemetry   telemetry.TelemetryService
-	launcher    rtc.EgressLauncher
-	shutdown    chan struct{}
+	client           rpc.EgressInternalClient
+	clientDeprecated egress.RPCClient
+	useNewRPC        bool
```

Comment:
Since this is probably going to stick into the code for a while because of the OSS version, could we use a name a little bit more specific than "new" and "deprecated"? This usually ends up turning into "useNewNewRPC" some months later... usePsRPC?